swagger.yml

```diff
@@ -467,6 +467,9 @@ definitions:
     properties:
       openstack:
         $ref: '#/definitions/OpenstackSpec'
+      audit:
```

Comment:
if we make this an enum, the docs will tell what values are valid as we do for backup: https://github.com/sapcc/kubernikus/blob/master/swagger.yml#L526
wdyt?